plugins/org.komodo.relational/src/org/komodo/relational/json/JsonVisitor.java

```diff
@@ -224,54 +259,131 @@ private String id(KomodoObject kObject) throws KException {
         return transaction.decode(kObject.getName(transaction));
     }
 
-    private boolean ignoreProperty(String propertyName) {
-        if (propertyName.startsWith(JcrLexicon.Namespace.PREFIX))
+    private boolean hasPrefix(String attributeName) {
+        return attributeName.matches(PREFIX_PATTERN);
+    }
+
+    private boolean ignoreAttribute(String attributeName) {
+        if (attributeName.startsWith(JcrLexicon.Namespace.PREFIX))
             return true; // ignore built-in properties like primaryType and mixinTypes
 
-        if (propertyName.startsWith(ModeShapeLexicon.Namespace.PREFIX))
+        if (attributeName.startsWith(ModeShapeLexicon.Namespace.PREFIX))
             return true; // ignore built-in properties like mode:sha1
 
-        if (propertyName.equals(VdbLexicon.Vdb.ORIGINAL_FILE))
+        if (attributeName.equals(VdbLexicon.Vdb.ORIGINAL_FILE))
             return true;
 
-        if (propertyName.equals(VdbLexicon.Model.MODEL_DEFINITION))
+        if (attributeName.equals(VdbLexicon.Model.MODEL_DEFINITION))
             return true;
 
-        if (propertyName.equals(StandardDdlLexicon.DDL_EXPRESSION))
+        if (attributeName.equals(STAR))
             return true;
 
-        if (propertyName.equals(StandardDdlLexicon.DDL_LENGTH))
+        if (attributeName.equals(StandardDdlLexicon.DDL_EXPRESSION))
```

Comment:
Would it be better just to check to see if the property prefix starts with StandardDdlLexicon.Namespace.PREFIX ("ddl:")? Or do you have to enumerate since some you don't want to ignore?